render.clj

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bb
+
+(require '[clojure.tools.cli :refer [parse-opts]])
+
+(def cli-options
+  [[nil "--mac-sha SHA" "The sha256 of the mac artifact"
+    :default ""
+    :missing "--mac-sha must be specified"]
+   [nil "--linux-sha SHA" "The sha256 of the linux artifact"
+    :default ""
+    :missing "--linux-sha must be specified"]
+   [nil "--version VERSION" "The new version to publish"
+    :default ""
+    :missing "--linux-sha must be specified"]
+   [nil "--template" "Path of the template file"
+    :default "clojure-lsp-native.rb.tmpl"]])
+
+(let [{:keys [version linux-sha mac-sha template]} (-> (parse-opts *command-line-args* cli-options)
+                                                       :options)
+      template (slurp template)]
+  (format template version version mac-sha version linux-sha))
```

Comment:
Maybe use https://clojuredocs.org/clojure.string/replace?
soething like:
```clojure
(-> template
     (string/replace "<mac-sha>" mac-sha)
     (string/replace "<linux-sha>" linux-sha))
```

Reply:
That works too, it's a matter of which poison you prefer I suppose :-)

Reply:
Yeah, both works to me :)

Reply:
I suppose regexes will break less, I'll make the change